tests/test_io/test_3brain.py

```diff
@@ -6,7 +6,7 @@
 
 from probeinterface import read_3brain
 
-from probeinterface.testing import validate_probe_dict
+from ..utilities import validate_probe_dict
```

Comment:
```suggestion
from probeinterface.testing import validate_probe_dict
```
@musicinmybrain can you elaborate on why it's not a good ide to include the `validate_probe_dict` in the API? I think it's an important function also for the package, so I would leave it there...